.github/workflows/pr-highlight-changes.yml

```diff
@@ -22,7 +22,7 @@ jobs:
               run: |
                   npm install -g pnpm@7
                   npm -g i @wordpress/env@5.1.0
-                  pnpm install --filter code-analyzer --filter cli-core
+                  pnpm install --filter monorepo-utils --filter code-analyzer --filter cli-core
```

Comment:
@ObliviousHarmony This was the change required to get this passing. The `postinstall` event was being fired which builds `monorepo-utils` but dependencies weren't installed.

Reply:
I see! Sounds good then!